Gemfile

```diff
@@ -10,6 +10,7 @@ gem 'bootsnap', '>= 1.4.4', require: false
 gem 'bulma_form_builder'
 gem 'bulma-rails', '~> 0.9.3'
 gem 'devise', '~> 4.8', '>= 4.8.1'
+gem 'kaminari', '~> 1.2', '>= 1.2.2'
```

Comment:
FYI, recently gem [pagy](https://github.com/ddnexus/pagy) is also very popular for pagination